server/mongo-init.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+echo '========================================================================='
```

Comment:
This is not required and happens by default... I removed all containers / images and db-data and was able to start up everything without this script.

The main fix that is needed is this:
* An init script on the API container that waits to see the message `waiting for connections on port 27017` before starting itself.

Reply:
For the waiting connections I made a PR #39

Reply:
> The main fix that is needed is this:
> 
> * An init script on the API container that waits to see the message `waiting for connections on port 27017` before starting itself.

See my last comment below.